packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -144,7 +156,7 @@ const PostsPage = ({post, refetch, classes}: {
         <AnalyticsContext pageSectionContext="postHeader"><div className={classes.title}>
           <div className={classes.centralColumn}>
             {commentId && <CommentPermalink documentId={commentId} post={post} />}
-            {post.eventImageId && <div className={classes.headerImageContainer}>
+            {post.eventImageId && <div className={classNames(classes.headerImageContainer, commentId ? classes.headerImageContainerWithComment : '')}>
```

Comment:
Can get rid of that ternary by using the other classNames style.

```js
classNames(class1, {class2: booleanVar})
```

[More examples](https://www.npmjs.com/package/classnames).